model/src/main/java/org/alfresco/transform/common/RequestParamMap.java

```diff
@@ -19,15 +19,22 @@
  * <http://www.gnu.org/licenses/lgpl-3.0.html>.
  * #L%
  */
-package org.alfresco.transform.client.util;
+package org.alfresco.transform.common;
 
-import org.alfresco.transform.client.model.config.CoreVersionDecorator;
+import org.alfresco.transform.config.CoreVersionDecorator;
 
 /**
  * Request parameters and transform options used in the core transformers.
  */
 public interface RequestParamMap
 {
+    // html parameter names

```

Comment:
This class should not be an interface if it holds conststants.

Reply:
I agree, however it is extended in the deprecated t-base used by customers, so risky to change. We can once we have don the fork and removed the deprecated t-base.